File: src/main/java/pl/yalgrin/playnite/simplesync/mapper/objects/SourceMapper.java
package pl.yalgrin.playnite.simplesync.mapper.objects;

import org.springframework.stereotype.Component;
import pl.yalgrin.playnite.simplesync.library.domain.Source;
import pl.yalgrin.playnite.simplesync.library.dto.SourceDTO;

@Component
public class SourceMapper extends AbstractObjectMapper<Source, SourceDTO> {

    @Override
    protected SourceDTO createDTO() {
        return new SourceDTO();
    }
}

File: src/main/kotlin/pl/yalgrin/playnite/simplesync/library/mapper/SourceMapper.kt
package pl.yalgrin.playnite.simplesync.library.mapper

import org.springframework.stereotype.Component
import pl.yalgrin.playnite.simplesync.library.domain.Source
import pl.yalgrin.playnite.simplesync.library.dto.SourceDTO

@Component
class SourceMapper : LibraryObjectMapperImpl<Source, SourceDTO>() {
    override fun createDTO(): SourceDTO = SourceDTO()
}